impeller/aiks/BUILD.gn

```diff
@@ -67,28 +67,60 @@ impeller_component("aiks_playground") {
   ]
 }
 
-impeller_component("aiks_unittests") {
+impeller_component("context_spy") {
   testonly = true
 
   sources = [
-    "aiks_unittests.cc",
-    "canvas_recorder_unittests.cc",
-    "canvas_unittests.cc",
     "testing/context_mock.h",
     "testing/context_spy.cc",
     "testing/context_spy.h",
-    "trace_serializer_unittests.cc",
   ]
   deps = [
-    ":aiks",
-    ":aiks_playground",
-    "//flutter/impeller/geometry:geometry_asserts",
-    "//flutter/impeller/golden_tests:golden_playground_test",
-    "//flutter/impeller/playground:playground_test",
-    "//flutter/impeller/scene",
-    "//flutter/impeller/typographer/backends/stb:typographer_stb_backend",
+    "//flutter/impeller/renderer",
     "//flutter/testing:testing_lib",
-    "//flutter/third_party/txt",
+  ]
+}
+
+template("aiks_unittests_component") {
```

Comment:
This is the only way I could figure out how to share these files.  The way the golden tests works is that they have to be compiled with a specific define.